src/dotnet/Mgmt.CI.BuildTools/CI/CI.BuildTasks/Tests/CI.BuildTasks.Tests/Tests.CI.BuildTasks.csproj

```diff
@@ -4,7 +4,11 @@
     <Description>Tests for MS.Az.NetSdk.Build.Tasks msbuild build tasks tests</Description>
     <AssemblyTitle>MS.Az.NetSdk.Build.Tasks Tests</AssemblyTitle>
     <AssemblyName>Build.Tasks.Tests</AssemblyName>
-  </PropertyGroup>
+  </PropertyGroup>  
+  <PropertyGroup>
+		<TargetFramework>netcoreapp2.1</TargetFramework>
+    <IsPackable>false</IsPackable>
```

Comment:
While it is fine to put in the individual packages you could have also put them into the common props file.